attachments/models.py

```diff
@@ -229,7 +230,10 @@ class Attachment(models.Model):
 
     file = models.FileField(_("file"), upload_to=get_attachment_dir,
                             max_length=255)
-    content_type = models.ForeignKey(ContentType)
+    content_type = models.ForeignKey(
+        ContentType,
+        on_delete=models.CASCADE,
```

Comment:
Django 2.2 requires all FKs define `on_delete`. Kind of annoying, but whatever. `CASCADE` is what django 1.11 is doing by default.